app/utils/whitelabel.ts

```diff
@@ -89,7 +92,7 @@ services and re-entry programs.`,
   aboutPageTitle: "SF Service Guide",
   enabledTranslations: ["en", "es", "tl", "zh-TW"],
   homePageComponent: "HomePage",
-  intercom: false,
```

Comment:
Was intercom showing up on the Our 415 white label? I would have thought it wouldn't since the SF Families whitelabel should have inherited the `false` value from here. Anyhow, I don't think we want to set intercom to `true` by default, or if we do, we will also need to set intercom to `false` on UCSF's whitelabel

Reply:
Oh no, I thought I had removed this - this was not intended to be included. Thanks for catching it.

Reply:
Pushed up the fixup!